shared/components/ember-output.jsx

```diff
@@ -0,0 +1,90 @@
+/* global React */
+
+export default React.createClass({
+    displayName: 'EmberOutput',
+
+    propTypes: {
+        source  : React.PropTypes.string.isRequired,
+        context : React.PropTypes.object.isRequired
+    },
+
+    injectMessages: function (locale, messages) {
+        var container  = this.container;
+        var lookupName = locale.toLowerCase();
+
+        if (container.has('locale:' + lookupName)) {
+            container.unregister('locale:' + lookupName);
+        }
+
+        container.register('locale:' + lookupName, {
+            locale:   locale,
+            messages: messages
+        }, {
+            instantiate: false
+        });
```

Comment:
>

> Is this something an Ember dev would understand?

Depending on the developer :)  Typically you don't have to interface with the container but in our case  since the messages are POJOs passed into the react component I need to pass them along to Ember.  

Just to be clear, this is not something consumers of ember-intl need to do.  This is the "glue" between React and the Ember app.